qa/standalone/ceph-helpers.sh

```diff
@@ -120,6 +120,13 @@ function setup() {
     teardown $dir || return 1
     mkdir -p $dir
     mkdir -p $(get_asok_dir)
+    if [ $(ulimit -n) -le 1024 ];then
+        ulimit -n 4096 || return 1
+        if [ $(ulimit -n) -ne 4096 ];then
+            echo "Cannot setup ulimit to 4096 !"
```

Comment:
i am curious under what condition `ulimit -n 4096` returns 0 but fails to do so?

Reply:
make sense. fixed.

Reply:
[~] wjw@cephdev >  ulimit -n
943038

So I guess that makes FreeBSD safe for the time being

Reply:
Thx @wjwithagen !